XRTK.SDK/Packages/com.xrtk.sdk/StandardAssets/Materials/Wireframe.mat

```diff
@@ -8,13 +8,14 @@ Material:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: Wireframe
-  m_Shader: {fileID: 4800000, guid: 4a2ab4faa2fc4a009bcfdf058d1e0659, type: 3}
+  m_Shader: {fileID: 4800000, guid: 5c1653eb4e20b76499141de7bc57c063, type: 3}
```

Comment:
We're assigning a whole new shader to this material?

Reply:
Yes, replacing it with the new MRTK Wireframe shader, which does compile on Android.
It doesn't work and can't be used, but now it doesn't stop the entire project from being built